packages/bits/src/lib/expander/expander.component.less

```diff
@@ -1,8 +1,15 @@
 @import (reference) "../../styles/mixins/expander-base";
+@import (reference) "../../styles/mixins/focus.less";
 
 .nui-expander {
     .expander();
 
+    &__header {
+        &:focus {
+            .focus-outline();
```

Comment:
I wonder if we should avoid using this mixin until the setting of the z-index is removed:

https://github.com/solarwinds/nova/blob/3bffdc91025981eee619e3bafb53aefb993fc05e/packages/bits/src/styles/mixins/focus.less#L3-L4

To be safe, maybe we should just duplicate those styles here without the z-index.

By the way, I'm not sure why the mixin has an outline rule with a value of `none`, but chrome reads that as an invalid value for outline. However, chrome does like `0` for that rule value. See this comment here for reference:

https://github.com/solarwinds/nova/pull/310#discussion_r641789686

Reply:
@Blakesalot `outline: none` looks pretty valid to me
![image](https://user-images.githubusercontent.com/22597043/122032283-26a66080-cdd8-11eb-9806-268c185c0928.png)

Reply:
Okay, thanks for checking. I'm not sure what I saw then :)

Reply:
@dmytrofirsov, aside from that, are you considering whether or not to use the mixin since it sets a z-index?

Reply:
I decided to make changes with a comment in the code